lang/test/org/partiql/lang/eval/evaluatortestframework/EvaluatorTestCase.kt

```diff
@@ -97,6 +102,21 @@ data class EvaluatorTestCase(
     /** This will show up in the IDE's test runner. */
     override fun toString() = when {
         groupName != null -> "$groupName : $query"
-        else -> "$query"
+        else -> query
+    }
+
+    /** A generated and human-readable description of this test case for display in assertion failure messages. */
+    fun testDetails(note: String, actualResult: String? = null): String {
```

Comment:
Missing `target` in the `testDetails` generated string

Reply:
done.